css/_language-picker.scss

```diff
@@ -1,16 +1,24 @@
 .language-picker {
-    background-color: rgba(255, 255, 255, 0.6);
+    background-color: rgba(255, 255, 255, 0.84);
     color: #090C0E;
     position: relative;
-    padding: 12px;
+    padding: 12px 8px;
     transition: background-color var(--transition-out), color var(--transition-out);
     min-width: 216px;
 
     display: -ms-inline-grid; display: inline-grid;
-    -ms-grid-columns: -webkit-max-content 32px;
-    -ms-grid-columns: max-content 32px;
-    grid-template-columns: -webkit-max-content 32px;
-    grid-template-columns: max-content 32px;
+    -ms-grid-columns: -webkit-max-content 16px;
+    -ms-grid-columns: max-content 16px;
+    grid-template-columns: -webkit-max-content 16px;
+    grid-template-columns: max-content 16px;
+
+    @media (min-width: 360px) {
+        padding: 12px;
+        -ms-grid-columns: -webkit-auto 32px;
```

Comment:
Ik zie veel terugkerende pixel units, (en los van de vraag of we daar niet liever em/rem units van zouden maken), is het goed om hier misschien een variabele van te maken omdat we scss gebruiken? :)